code/style.css

```diff
@@ -1,7 +1,226 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+@import url("https://fonts.googleapis.com/css2?family=Lora:wght@400;600;700&family=Poiret+One&display=swap");
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  color: #000;
+  font-family: "Poiret One", sans-serif;
+}
+
+.hero-image {
+  height: 80vh;
+  width: 100%;
+  background-image: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.2)),
+    url(./pictures/blue-art.jpg);
+  background-position: top center;
+  background-size: cover;
+} 
+
+.text-container{
+  padding-top: 10vh;
+  text-align: center;
+  color: #F5F1E8;
+}
+
+.hero-image h1 {
+  font-size: 13vh;
```

Comment:
If it wasn't for this font-size, your page would be responsive for mobile devices too. With this font-size it gets a bit wonky bellow 500px. But it wasn't a requirement for this project, so this is just me nitpicking 🙈